dts/bindings/wifi/infineon,airoc-wifi-common.yaml

```diff
@@ -0,0 +1,20 @@
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
+  wifi-reg-on-gpios:
+    description: |
+      Power-up/down gpio to control the internal regulators used
+      by the WiFi section of AIROC Wi-Fi device.
+    type: phandle-array
+
+  wifi-host-wake-gpios:
```

Comment:
can be renamed to `interrupt` IMO

Reply:
"WIFI_HOST_WAKE" comes from the chip datasheet, and that's how Infineon defined the property.  I didn't want to change it.

Reply:
yeah, same is called `interrupt` in Linux Kernel context, so was my suggestion. prefix `wifi-` is implicit already, so can be removed

Reply:
The names:
- wifi_host_wake
- wifi_reg_on
- bt_reg_on
are already in use in the Infineon WiFi and Bluetooth drivers, and match names used throughout modules/hal/infineon and zephyr/modules/hal_infineon, as well as following the naming convention used in their hardware datasheets.  Since I'm not part of Infineon, and I'm only adding a new capability on top of their existing driver, I think it's not my role to modify their names.